resources/assets/js/components/Application/Review/ApplicationPreview.tsx

```diff
@@ -729,7 +677,7 @@ const ApplicationPreview: React.FunctionComponent<ApplicationPreviewProps> = ({
             data-c-align="base(center) tp(right)"
           >
             <a
-              href="https://talent.test/demo/application-07"
+              href="https://talent.test/demo/application-07" // To
```

Comment:
Does this need to be updated for the Fit page route?

Reply:
Yep! I've made a new issue for fixing urls for the application https://github.com/GCTC-NTGC/TalentCloud/issues/4647